proto/config/netconfig.proto

```diff
@@ -181,6 +187,22 @@ message CellularAccessPoint {
   // Not listed technologies will not be tried.
   // If empty, then modem will select RAT automatically.
   repeated org.lfedge.eve.common.RadioAccessTechnology preferred_rats = 7;
+  // The IP addressing type to use for the default bearer.
+  org.lfedge.eve.common.CellularIPType ip_type = 8;
+  // Access Point Name (APN) for the attach (initial) bearer.
+  // This field is optional. If not specified, EVE will not send any attach bearer
+  // configuration to the modem. Instead, the modem will use one of its pre-installed
+  // profiles (see CellularProfile) to determine the configuration.
+  string attach_apn = 9;
+  // The IP addressing type to use for the attach bearer.
+  // Leave undefined if attach_apn is not specified.
+  org.lfedge.eve.common.CellularIPType attach_ip_type = 10;
+  // Authentication protocol used for the attach bearer.
+  // Leave undefined if attach_apn is not specified.
+  CellularAuthProtocol attach_auth_protocol = 11;
+  // Encrypted user credentials for the attach bearer (if required).
+  // Leave empty if attach_apn is not specified.
+  org.lfedge.eve.common.CipherBlock attach_cipher_data = 12;
```

Comment:
@eriknordmark Now I'm thinking if it was a good approach to add another CipherBlock field for attach username/password, or whether I should have used the already defined `org.lfedge.eve.common.CipherBlock cipher_data = 4` and add `cellular_net_attach_username` and `cellular_net_attach_password` next to [existing cellular fields](https://github.com/lf-edge/eve-api/blob/main/proto/evecommon/acipherinfo.proto#L81-L84C10) in the EncryptionBlock.
Which approach is preferred?
We can still change this, it is not yet used.

Reply:
I guess a single cipherblock in the object avoids some complexity since in theory each cipherblock can refer to a different controller encryption certificate, thus one can fail and the other succeed when there is some missing cert.